### PR TITLE
pass payload version and manifests to kas and kcm

### PR DESCRIPTION
**Pull request by deads2k**
Passes the feature gate and featureset values used for rendering via the FeatureGate manifest contained in rendered manifests.  The dependencies in operators should now be complete.

### CODE REVIEW

**Comment by deads2k** [comment]
found a trip dependency in https://github.com/openshift/cluster-kube-apiserver-operator/pull/1490

I'll need to re-test this after merging it

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -199,7 +196,8 @@ then
 		--cluster-config-file=/assets/manifests/cluster-network-02-config.yml \
 		--cluster-auth-file=/assets/manifests/cluster-authentication-02-config.yaml \
 		--infra-config-file=/assets/manifests/cluster-infrastructure-02-config.yml \
-		"${ADDITIONAL_FLAGS[@]}"
+		--rendered-manifest-files=/assets/manifests \
+		--payload-version=$VERSION
```
Comment (JoelSpeed) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:200]: Do we not need the same for KCMO below?

Comment (deads2k) [reply data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:200]: now updated.

**Comment by deads2k** [comment]
build failures

/retest

**Comment by deads2k** [comment]
build failures

/retest

**Pull request merged**